contributors/devel/sig-architecture/vendor.md

```diff
@@ -164,10 +164,12 @@ Additionally:
 - If this is all good, approve, but don't LGTM, unless you also do code review
   or unless it is trivial (e.g. moving from k/k/pkg/utils -> k/utils).
 
+Licenses for dependencies are specified by the CNCF [Allowlist Policy].
```

Comment:
Should `[Allowlist Policy]` be a link to some external reference?

Reply:
Fixed :+1: